sql/core/src/test/scala/org/apache/spark/sql/CollationSQLExpressionsSuite.scala

```diff
@@ -2319,6 +2319,57 @@ class CollationSQLExpressionsSuite
     )
   }
 
+  // scalastyle:off nonascii
```

Comment:
please note that the entire suite is already wrapped in this scalastyle exemption, so we can remove this here

https://github.com/apache/spark/blob/0e07873d368fa17dfff11e28fd45531f1f388864/sql/core/src/test/scala/org/apache/spark/sql/CollationSQLExpressionsSuite.scala#L35